src/Microsoft.Azure.WebJobs.Host/Scale/TriggerMetadata.cs

```diff
@@ -15,30 +11,31 @@ namespace Microsoft.Azure.WebJobs.Host.Scale
     /// </summary>
     public class TriggerMetadata
     {
-        private readonly List<object> _properties;
+        public TriggerMetadata(string functionName, JObject metadata)
```

Comment:
I revamped the trigger metadata / provider APIs to make them cleaner. We may need to iterate more.